.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Flux Accounting Developer Environment",
+  "dockerFile": "Dockerfile",
+  "context": "../",
+
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.defaultProfile.linux": "bash"
+  },
+      "extensions": [
+         "ms-vscode.cmake-tools"
+      ]
+    }
+  },
+  "postStartCommand": "git config --global --add safe.directory /workspaces/flux-accounting"
+}
```

Comment:
does it matter if this line does or doesn't have a newline at the end of it?

Reply:
It does not - it looks like json but actually it breaks many rules of it. I tested this fully and the newline does not seem to make a difference.

Reply:
hey you have a new picture! I like it!

Reply:
thanks! my hair is a lot longer now, i don't look much like that graduation photo from 2019 anymore 😆 feel free to set MWP on this whenever you are ready

Reply:
Done! Thanks @cmoussa1